github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Lighthouse
+        run: npm install -g @lhci/cli@0.7.x
+      - name: Lighthouse Report
+        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
+  webhint:
+    name: Webhint
+    runs-on: ubuntu-22.04
+    steps:
```

Comment:
Great job @Ibtisam34 🤝

- [ ] You did well adding this `linters.yml` file. However, there are other two linter files still missing in your project. The `.hintrc`, and `stylelintrc.json` files. You can find these files in the [Linter files](https://github.com/microverseinc/linters-config/tree/master/html-css) link. Kindly add these two files to the root directory of your project.